visualizer/options-menu.js

```diff
@@ -3,13 +3,39 @@
 const HtmlContent = require('./html-content.js')
 const d3 = require('./d3.js')
 
+function getCodeAreaDescription (area) {
```

Comment:
I moved this here to allow generating descriptions for all the dependency code areas in the future. Will prob move it to ui.js so it's in the same place as the getLabelKey() (i forget the name) one